From 61592d46e7fc5644c4b5148c7ca6bf767131e504 Mon Sep 17 00:00:00 2001
From: okozachenko1203 <okozachenko1203@gmail.com>
Date: Fri, 31 Mar 2023 23:41:43 +1100
Subject: [PATCH] Update chart.metadata.version to reflect breaking change in
 helm v3.5.2

https: //github.com/helm/helm/issues/9342
Change-Id: I1dbe7b0b85380e713ebb5dcdd7ecbfc6a438b852
(cherry picked from commit ebee3263b6b3d3fa213ea8f837911b89785a4700)
---
 .../templates/kubernetes/fragments/install-helm-modules.sh    | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/magnum/drivers/common/templates/kubernetes/fragments/install-helm-modules.sh b/magnum/drivers/common/templates/kubernetes/fragments/install-helm-modules.sh
index 475e8dbf6c..a0b3f4bc75 100644
--- a/magnum/drivers/common/templates/kubernetes/fragments/install-helm-modules.sh
+++ b/magnum/drivers/common/templates/kubernetes/fragments/install-helm-modules.sh
@@ -72,8 +72,8 @@ else
         cat << EOF > Chart.yaml
 apiVersion: v1
 name: magnum
-version: metachart
-appVersion: metachart
+version: 1.0.0
+appVersion: v1.0.0
 description: Magnum Helm Charts
 EOF
         sed -i '1i\dependencies:' requirements.yaml
